packages/@webex/plugin-presence/package.json

```diff
@@ -31,7 +31,8 @@
     "@webex/test-helper-test-users": "workspace:*",
     "eslint": "^8.24.0",
     "prettier": "^2.7.1",
-    "sinon": "^9.2.4"
+    "sinon": "^9.2.4",
```

Comment:
We should remove the caret from existing dependencies as well.

cc- @Shreyas281299